integration_tests/integration/home.cy.ts

```diff
@@ -149,33 +148,7 @@ context('Home page', () => {
       expect(loc.pathname).to.eq(adjudicationUrls.confirmDISFormsIssued.root)
     })
   })
-  it('link should not exist to the correct location - adjudication data', () => {
-    cy.visit(adjudicationUrls.homepage.root)
-    const homepage: HomepagePage = Page.verifyOnPage(HomepagePage)
-    homepage.dataInsightsLink().should('not.exist')
-  })
-})
-
-context('Home page for RNI - Ranby (HMP)', () => {
-  beforeEach(() => {
-    cy.task('reset')
-    cy.task('stubSignIn', [
-      {
-        caseLoadId: 'RNI',
-        description: 'Ranby (HMP)',
-        currentlyActive: true,
-        type: '',
-        caseloadFunction: '',
-      },
-    ] as CaseLoad[])
-    cy.task('stubAuthUser', {
-      username: 'USER1',
-      activeCaseLoadId: 'RNI',
-    })
-    cy.signIn()
-  })
-
-  it('should link to the correct location - adjudication data', () => {
+  it('should link to the correct location - data insights', () => {
```

Comment:
As we're now enabling it for all caseloads, moved this test to be within `context('Home page'`